### DISCUSSION TITLE
私钥格式似不支持「DES-EDE3-CBC」？不知是否有开发计划？

### DISCUSSION THREAD

**Discussion by sicifus** [lollipopkit/flutter_server_box#342] [General]
不确定是bug还是功能缺失，暂先在这个版块发一下，L大若觉得不合适我再去issue去提。
输入这个格式的私钥，报错信息如图所示。

相同的私钥在DaRemote下能正常保存（非引战，只是表示私钥没有录错）

**Comment by sicifus** [comment]
![Uploading 捕获.JPG…]()
补图

**Comment by lollipopkit** [comment]
这是第三方库没有实现，我暂无能力解决该问题，抱歉

**Comment by GT-610** [comment]
DES已弃用，原因见 #432 。
